### PR TITLE
*: add tidb_enable_stats_owner

### PR DESCRIPTION
**Pull request by hawkingrei**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:

https://github.com/pingcap/docs-cn/pull/18817

- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -1976,6 +1976,15 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
 - This variable controls whether the corresponding TiDB instance can become a DDL owner or not. If there is only one TiDB instance in the current TiDB cluster, you cannot prevent it from becoming a DDL owner, which means you cannot set it to `OFF`.
 
+### tidb_enable_stats_owner <span class="version-mark">New in v8.4.0</span>
+
+- Scope: GLOBAL
+- Persists to cluster: No, only applicable to the current TiDB instance that you are connecting to.
+- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
+- Default value: `ON`
+- Possible values: `OFF`, `ON`
+- This variable controls whether the corresponding TiDB instance can become a stats owner or not. If there is only one TiDB instance in the current TiDB cluster, you cannot prevent it from becoming a stats owner, which means you cannot set it to `OFF`.
```
Comment (songrijie) [inline system-variables.md:1986]:
```suggestion
- This variable controls whether the corresponding TiDB instance is able to run [automatic update of optimizer statistics](/statistics.md#automatic-update). If there is only one TiDB instance in the current TiDB cluster, you cannot prevent it from running automatic update of optimizer statistics , which means you cannot set it to `OFF`.
```

**Review by songrijie — APPROVED** [review]
rest LGTM

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: tidb-configuration-file.md [hunk 2]
```diff
@@ -951,6 +951,13 @@ Configuration items related to read isolation.
 - The value of this configuration will initialize the value of the system variable [`tidb_enable_ddl`](/system-variables.md#tidb_enable_ddl-new-in-v630)
 - Before v6.3.0, this configuration is set by `run-ddl`.
 
+### `tidb_enable_stats_owner` <span class="version-mark">New in v8.4.0</span>
+
+- This configuration controls whether the corresponding TiDB instance can become a stats owner or not.
```
Comment (Oreoxmt) [inline tidb-configuration-file.md:956]:
```suggestion
- This configuration controls whether the corresponding TiDB instance whether the corresponding TiDB instance can run [automatic statistics update](/statistics.md#automatic-update) tasks.
```

Comment (Oreoxmt) [reply tidb-configuration-file.md:956]: @hawkingrei PTAL

File: tidb-configuration-file.md [hunk 3]
```diff
@@ -951,6 +951,13 @@ Configuration items related to read isolation.
...
+- Default value: `true`
+- Possible values: `OFF`, `ON`
```
[the remark this answers is not in the document]
Comment (Oreoxmt) [reply tidb-configuration-file.md:958]: @hawkingrei PTAL

File: tidb-configuration-file.md [hunk 4]
```diff
@@ -951,6 +951,13 @@ Configuration items related to read isolation.
 - The value of this configuration will initialize the value of the system variable [`tidb_enable_ddl`](/system-variables.md#tidb_enable_ddl-new-in-v630)
 - Before v6.3.0, this configuration is set by `run-ddl`.
 
+### `tidb_enable_stats_owner` <span class="version-mark">New in v8.4.0</span>
+
+- This configuration controls whether the corresponding TiDB instance whether the corresponding TiDB instance can run [automatic statistics update](/statistics.md#automatic-update) tasks.
```
Comment (qiancai) [inline tidb-configuration-file.md:956]:
```suggestion
- This configuration controls whether the corresponding TiDB instance can run [automatic statistics update](/statistics.md#automatic-update) tasks.
```

File: tidb-configuration-file.md [hunk 5]
```diff
@@ -951,6 +951,13 @@ Configuration items related to read isolation.
...
+- Default value: `true`
+- Possible values: `true`, `false`
+- The value of this configuration will initialize the value of the system variable [`tidb_enable_stats_owner`](/system-variables.md#tidb_enable_stats_owner-new-in-v840)
```
Comment (qiancai) [inline tidb-configuration-file.md:959]:
```suggestion
- The value of this configuration will initialize the value of the system variable [`tidb_enable_stats_owner`](/system-variables.md#tidb_enable_stats_owner-new-in-v840).
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**